content/post/lapjv.md

```diff
@@ -441,8 +441,8 @@ A \\cdot \\vec{x} \\leq \\vec{b}
 \\vec{x} \\geq 0
 \\end{equation}
 We minimize the scalar product of \\(\\vec{c}\\) and \\(\\vec{x}\\) given the set
-of linear inequations depending of \\(\\vec{x}\\) and the requirement that all
```

Comment:
I had never heard of inequations before, so I assumed you meant linear equations. I hope I am right ;) @vmarkovtsev